packages/snjs/lib/services/api/api_service.ts

```diff
@@ -113,104 +151,117 @@ export class SNApiService extends PureService {
     return this.session;
   }
 
-  private async path(path: string) {
-    const host = await this.getHost();
-    if (!host) {
-      throw `Attempting to build path ${path} with no host.`;
-    }
-    if (!path) {
-      throw 'Attempting to build path with null path.';
-    }
-    return joinPaths(host, path);
-  }
-
-  private get apiVersion() {
```

Comment:
Needed for this test: https://github.com/standardnotes/snjs/blob/master/test/keys.test.js#L820

Reply:
Thanks! I fixed it by making `V0_API_VERSION` a static class variable.